lib/DDG/Meta/Data.pm

```diff
@@ -92,15 +92,19 @@ unless(%ia_metadata){
             $ia_metadata{id}{$id} = $module_data;
             #add new ia to ia_metadata{module}. Multiple ias per module possible
             push @{$ia_metadata{module}{$perl_module}}, $module_data;
-            # by language for multilang wiki
+
             if($is_fathead){
+                my $source = $module_data->{src_id};
                 # by source number for fatheads
-                $ia_metadata{source}{$module_data->{src_id}} = $module_data;
+                $ia_metadata{source}{$source} = $module_data;
```

Comment:
We might want to make these keys "fathead_source", "fathead_lang", etc.  Not a show stopper though.